### PR TITLE
Fix Dateutilityservice Logic Issue 169

### LINKED ISSUE #169: dateUtilityService -
Line 8 - 
`Condition 'date === 'Invalid Date'' is always false at this point because the else branch of the condition 'typeof date !== 'object'' at line 8 has been taken.`

See: https://deepscan.io/dashboard/#view=project&pid=2129&bid=10806&subview=issues

Solution:
remove `date === 'Invalid Date'`

### PR DESCRIPTION
**Pull request by martypowell**
Resolves #169 

Removed unreachable condition.

### CODE REVIEW

File: js/apps/events-page/services/dateUtilityService.js [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 	const dateUtilityService = () => {
 		const addDays = (dateOrString, daysToAdd) => {
-			const date = typeof dateOrString === 'string' ? new Date(dateOrString) : dateOrString;
+			const date = typeof dateOrString === 'string' ?
+				new Date(dateOrString) :
+				dateOrString;
```
Comment (ascire) [inline js/apps/events-page/services/dateUtilityService.js]: Personally I prefer ternary notation at the beginning of the line but that's splitting hairs.

File: js/apps/events-page/services/dateUtilityService.js [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
 
-			if (typeof date !== 'object' || date === 'Invalid Date') {
-				return date;
-			}
+			if (typeof date !== 'object') return date;
```
Comment (ascire) [inline js/apps/events-page/services/dateUtilityService.js]: Is lint check OK with you removing curly braces from the if body?  I've had some style checks yell at me for doing this.

File: js/apps/events-page/services/dateUtilityService.js (hunk 1 above)
Comment (martypowell) [reply js/apps/events-page/services/dateUtilityService.js]: @ascire There was a reason i did it this way, and now I don't know why. I originally thought this was an error from the IDE, but I justed tested it and no dice. Mike prefers taht way too, so I am going to update our guidelines.

File: js/apps/events-page/services/dateUtilityService.js (hunk 2 above)
Comment (martypowell) [reply js/apps/events-page/services/dateUtilityService.js]: This is ok per airbnb - https://github.com/airbnb/javascript#blocks--braces

**Pull request merged**